examples/dt/bgp/bgp_dt01/edpm/networkers/values.yaml

```
@@ -0,0 +1,267 @@
+          edpm-networker-0:
+            nic2: 6d:fe:54:3f:8a:02  # CHANGEME
+          edpm-networker-1:
+            nic2: 6e:fe:54:3f:8a:02  # CHANGEME
+          edpm-networker-2:
+            nic2: 6f:fe:54:3f:8a:02  # CHANGEME
+        edpm_network_config_template: |
+          ---
+          {% set mtu_list = [ctlplane_mtu] %}
+          {% for network in nodeset_networks %}
+          {{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
+          {%- endfor %}
+          {% set min_viable_mtu = mtu_list | max %}
+          network_config:
+          - type: ovs_bridge
+            name: {{ neutron_physical_bridge_name }}
+            use_dhcp: false
+            dns_servers: {{ ctlplane_dns_nameservers }}
+            domain: {{ dns_search_domains }}
+            addresses:
+              - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_cidr }}
+            routes: []
+            members:
+              - type: interface
+                name: nic2
+                mtu: {{ min_viable_mtu }}
+                # force the MAC address of the bridge to this interface
+                primary: true
+          {% for network in nodeset_networks %}
+          {%   if not network.lower().startswith('bgp') %}
+              - type: vlan
+                mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+                vlan_id: {{ lookup('vars', networks_lower[network] ~ '_vlan_id') }}
+                addresses:
+                  - ip_netmask: >-
+                      {{
+                        lookup('vars', networks_lower[network] ~ '_ip')
+                      }}/{{
+                        lookup('vars', networks_lower[network] ~ '_cidr')
+                      }}
+                routes: {{ lookup('vars', networks_lower[network] ~ '_host_routes') }}
+          {% endif %}
+          {% endfor %}
+          - type: interface
+            name: nic3
+            use_dhcp: false
+            addresses:
+            - ip_netmask: {{ lookup('vars', 'bgpnet0_ip') }}/30
+          - type: interface
+            name: nic4
+            use_dhcp: false
+            addresses:
+            - ip_netmask: {{ lookup('vars', 'bgpnet1_ip') }}/30
+          - type: interface
+            name: lo
+            addresses:
+            - ip_netmask: {{ lookup('vars', 'bgpmainnet_ip') }}/32
+            - ip_netmask: {{ lookup('vars', 'bgpmainnetv6_ip') }}/128
+        edpm_nodes_validation_validate_controllers_icmp: false
+        edpm_nodes_validation_validate_gateway_icmp: false
+        edpm_enable_chassis_gw: true
+        edpm_selinux_mode: enforcing
+        edpm_sshd_allowed_ranges:
+          - 192.168.122.0/24
+        edpm_sshd_configure_firewall: true
+        enable_debug: false
+        gather_facts: false
+        neutron_physical_bridge_name: br-ex
+        neutron_public_interface_name: eth1
+        service_net_map:
+          nova_api_network: internalapi
+          nova_libvirt_network: internalapi
```

```suggestion
        edpm_sshd_allowed_ranges:
          - 192.168.122.0/24
        edpm_sshd_configure_firewall: true
        gather_facts: false
        neutron_physical_bridge_name: br-ex
        neutron_public_interface_name: eth1
```
Those removed above are no longer available and do nothing if included

---

done